source/administration/devops-command-center.rst

```diff
@@ -176,3 +184,17 @@ Channel Export
 --------------
 
 Please see the `Channel Export plugin documentation <https://mattermost.gitbook.io/channel-export-plugin>`_ for more information.
+
+Glossary
+--------
+
+- **Incident:** An event requiring the coordinated actions of one or more Mattermost users. An incident is either ongoing or closed.
+- **Playbook:** A task-based process that's followed in order to resolve an incident.
+- **Tasks:** The individual steps required to complete the stages of an incident. Tasks can optionally be assigned to specific incident participants into stages.
+- **Stage:** A set of tasks grouped together to achieve a specific goal of the workflow, which generally need to be completed before proceeding to the next stage of the incident resolution process.
+- **Commander:** The Mattermost user currently responsible for transitioning an incident from ongoing to closed.
+- **Incident channel:** A Mattermost channel dedicated to real-time conversation about the incident.
+- **Incident member:** A Mattermost user with access to the corresponding incident channel.
+- **The RHS:** The incident list and incident details displayed on the right hand side of the webapp. It is not available on mobile.
```

Comment:
Consider updating to clarify the meaning of RHS and remove potential ambiguity: "The incident list and incident details displayed on the right hand side (RHS) of the web app. The RHS is not available on mobile."